flintrock/scripts/download-hadoop.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
I'd rather reuse the existing Hadoop download script we have (`download-hadoop.py`) than add a new script alongside it. Can you do that?

Adding this script seems unnecessary.